setup.py

```diff
@@ -687,6 +687,7 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
 INSTALL_REQUIREMENTS = [
     'alembic>=1.2, <2.0',
     'argcomplete~=1.10',
+    'arrow==0.16.0',
```

Comment:
You should create a "plexus" extra and add "arrow" to the extra , rather than to install requirements.